sys-process/iotop/iotop-0.6_p20230124.ebuild

```
@@ -0,0 +1,37 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="ncurses(+)"
+
+MY_COMMIT="a14256a3ff74eeee59493ac088561f1bafab85a7"
+inherit distutils-r1 linux-info
+
+DESCRIPTION="Top-like UI used to show which process is using the I/O"
+HOMEPAGE="http://guichaz.free.fr/iotop/"
+SRC_URI="https://repo.or.cz/iotop.git/snapshot/${MY_COMMIT}.tar.gz -> ${P}.tgz"
```

Let's keep suffix the same:
```suggestion
SRC_URI="https://repo.or.cz/iotop.git/snapshot/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```